ivy_tests/test_ivy/test_frontends/test_paddle/test_random.py

```diff
@@ -7,6 +7,64 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# --- Helpers --- #
+# --------------- #
+
+
+@st.composite
+def _multinomial_helper(draw):
+    input_dtype_and_x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+            shape=helpers.get_shape(min_num_dims=1, max_num_dims=2, min_dim_size=2),
+        )
+    )
+    num_samples = draw(st.integers(min_value=1, max_value=10))
+    if num_samples > 2:
+        replacement = True
+    else:
+        replacement = draw(st.booleans())
+
+    input_dtype, x = input_dtype_and_x
+
+    total = sum(x)
+    x = [arr / total for arr in x]
```

Comment:
If the goal is to normalize the x values, I would recommend using min_value and max_value in  `helpers.dtype_and_values` method instead .

Reply:
That is what I did before but in order for the array to always sum to 1. We need to set `min_value` and `max_value` to `0.25` and `min_dim_size` and `max_dim_size` to 4. This means that the array will always be `[0.25 0.25 0.25 0.25]`. I think this isn't a good way to test the function as the array is static and limits it. So after generating the array with `helpers.dtype_and_values` I normalize it so the sum is 1.

Reply:
make sense to me!